_posts/en/newsletters/2020-07-08-newsletter.md

```diff
@@ -0,0 +1,227 @@
+  wallet, enter their seed or seed phrase, and recover any funds using
+  the wallet's default path.  But hardcoding particular paths also
+  constrains wallets to the uses envisioned by their own developers,
+  rather than allowing them to be used for other purposes or protocols.
+  Path templates provide a compact way for the user to describe to the
+  wallet what paths they want to use.  The compactness of path templates
+  makes it easy to backup the template along with the seed, helping
+  prevent users from losing funds.  An additional feature of the
+  proposed path templates is the ability to describe derivation limits
+  (e.g. that a wallet should derive no more than 50,000 keys in a
+  particular path), which can make it practical for a recovery procedure
+  to scan for bitcoins received to all possible wallet keys, eliminating
+  concerns about gap limits in HD wallets (see [Newsletter #52][news52
+  gap]).
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review Club][]
+meeting, highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting.*
+
+[Cache responses to `getaddr` to prevent topology leaks][review club #18991] is a
+[PR][Bitcoin Core #18991] by Gleb Naumenko that aims to make it more difficult
+for spy nodes to infer the P2P network topology using `addr` message gossiping.
+
+The discussion began by covering the basic concepts of address gossiping, and
+later focused on what privacy leaks are currently possible and what the PR
+intends to change.
+
+{% include functions/details-list.md
+  q0="What is the importance of `addr` relay and specifically the `getaddr`/`addr` protocol?"
+  a0="`addr` relay is used for nodes to find new potential peers on the P2P network."
+  a0link="https://bitcoincore.reviews/18991.html#l-34"
+
+  q1="Which properties of `addr` relay are important?"
+  a1="Nodes need to learn about a diverse set of peers with good uptime that were online recently."
+  a1link="https://bitcoincore.reviews/18991.html#l-57"
+
+  q2="How can a spy use `addr` messages to infer network topology?"
+  a2="There are potentially multiple ways to infer topology from `addr` messages, but the most discussed method was scraping nodes' address managers (addrman) to determine how an address record is spread across the network, and whether any nodes have a unique timestamp for that address record (indicating that they're probably directly connected to that address). This is the method that was used in the [Coinscope paper][]."
+  a2link="https://bitcoincore.reviews/18991.html#l-129"
+
+  q3="What could a malicious actor do if they were able to map the entire P2P topology?"
+  a3="Knowing the entire P2P network topology makes it easier to carry out attacks such as network partitions or [eclipse attacks][topic eclipse attacks]."
+  a3link="https://bitcoincore.reviews/18991.html#l-176"
```

Comment:
Hmm, is the goal of the trailing link to point to the best resource or to reference the part of the discussion that's being described in the Q&A?

Reply:
Since the review club discussion is linked to from the start, I'd reckon to optimise value for the reader (while still linking to a review club or optech resource in this section). Just one opinion.